test/api.test.ts

```diff
@@ -7,88 +7,53 @@ describe( 'api', () => {
 		const RealNow = Date.now;
 		const fakeNow = RealNow() + 24 * 60 * 1000;
 		Date.now = () => fakeNow;
+
 		afterAll( () => {
 			Date.now = RealNow;
 		} );
 
 		const EXPIRED_TIME = Date.now() - CONTAINER_EXPIRY_TIME - 1;
 		const GOOD_TIME = Date.now() - CONTAINER_EXPIRY_TIME + 1;
-		const images = [ { Image: getImageName( '1' ), Id: 1 }, { Image: getImageName( '2' ), Id: 2 } ];
+		const images = [
+			{ Image: getImageName( '1' ), Id: 1, Created: EXPIRED_TIME / 1000, Names: [ '/foo' ] },
+			{ Image: getImageName( '2' ), Id: 1, Created: EXPIRED_TIME / 1000, Names: [ '/bar' ] },
+		];
 
-		test( 'returns nothing for empty list of containers', () => {
-			expect( getExpiredContainers( [], () => 0 ) ).toEqual( [] );
+		afterEach( () => {
+			state.accesses = new Map();
+			state.containers = new Map();
 		} );
 
-		test( 'returns the whole list if everything is expired', () => {
-			const expiredImages = images.map( image => {
-				return Object.assign( {}, image, { Created: EXPIRED_TIME / 1000 } );
-			} );
-			expect( getExpiredContainers( expiredImages as any, () => CONTAINER_EXPIRY_TIME ) ).toEqual(
-				expiredImages
-			);
+		beforeEach( () => {
+			state.containers = new Map( images.map( image => [ image.Image, { ...image } ] ) as any );
 		} );
 
-		test( 'returns empty list if everything is before expiry', () => {
-			expect( getExpiredContainers( images as any, () => GOOD_TIME ) ).toEqual( [] );
+		test( 'returns nothing for empty list of containers', () => {
+			state.containers = new Map();
+			expect( getExpiredContainers() ).toEqual( [] );
 		} );
 
-		test( 'returns list of only images that have not expired', () => {
-			const getAccessTime = jest
-				.fn()
-				.mockReturnValueOnce( EXPIRED_TIME )
-				.mockReturnValueOnce( GOOD_TIME );
-			const oldImages = images.map( image => {
-				return Object.assign( {}, image, { Created: EXPIRED_TIME / 1000 } );
-			} );
-			expect( getExpiredContainers( oldImages as any, getAccessTime ) ).toEqual(
-				[].concat( oldImages[ 0 ] )
-			);
+		test( 'returns the whole list if everything is expired', () => {
+			expect( getExpiredContainers() ).toEqual( images );
 		} );
 
-		test( 'young images are not returned, regardless of access time', () => {
-			const expiredImages = images.map( image => {
-				return Object.assign( {}, image, { Created: EXPIRED_TIME / 1000 } );
-			} );
-			expiredImages[ 0 ].Created = Date.now() / 1000;
-			expect( getExpiredContainers( expiredImages as any, () => CONTAINER_EXPIRY_TIME ) ).toEqual(
-				[].concat( expiredImages[ 1 ] )
-			);
-		} );
-	} );
+		test.only( 'returns empty list if everything was accessed before expiry', () => {
+			state.accesses.set( 'foo', GOOD_TIME );
+			state.accesses.set( 'bar', GOOD_TIME );
 
-	describe( 'commitAccessTimes', () => {
```

Comment:
`getCommitAccessTime` is gone.

The equivalent functionality is already tested by the above test.